bbj-vscode/src/language/bbj-token-builder.ts

```diff
@@ -34,11 +34,11 @@ export class BBjTokenBuilder extends DefaultTokenBuilder {
                 LINE_BREAKS: true
             };
             return token;
-        } else if (terminal.name === 'LINE_BREAK') {
+        } else if (terminal.name === 'METHODRET_END') {
             const token: TokenType = {
                 name: terminal.name,
                 // Add more expceptional tokens here if an explicit line break token is needed
-                PATTERN: this.regexPatternFunction(/(?<=\r?\n[ \t]*methodret)([ \t]*(\r?\n|rem[ \t][^\n\r]*\r?\n))/i),
+                PATTERN: this.regexPatternFunction(/METHODRET[ \t]*(?=(;|\r?\n|rem[ \t]))/i),
```

Comment:
What is the `rem` part for?

Reply:
Just a leftover from when `REM` used to be a normal comment and not it's own statement.